### DISCUSSION TITLE
Validate value if value exists, otherwise do nothing

### DISCUSSION THREAD

**Discussion by nemonemi** [react-hook-form/react-hook-form#7181] [Q&A]
Hello,
How to apply a field validation if a date value is set, but not if it is not.

Mainly, the field is **not** mandatory, but if it is set, it should have a valid value.

**Comment by bluebill1049** [comment]
use `validate` function

**Reply by nemonemi** [reply]
Thanks for the quick response, @bluebill1049. The problem I have is that the `validate` method receives not a string, but a date object, and I cannot detect whether the value is missing or is invalid, because it always reports `Invalid date`.
